src/modules/menu.js

```diff
@@ -0,0 +1,86 @@
+import { isMobile, toggleClass, removeClass } from '../helpers/utils.js';
+
+export default () => {
+  // ░█▀▀▀ █── █▀▀ █▀▄▀█ █▀▀ █▀▀▄ ▀▀█▀▀ █▀▀
+  // ░█▀▀▀ █── █▀▀ █─▀─█ █▀▀ █──█ ──█── ▀▀█
+  // ░█▄▄▄ ▀▀▀ ▀▀▀ ▀───▀ ▀▀▀ ▀──▀ ──▀── ▀▀▀
+
+  const headerElement = document.querySelector('.header');
+  const menuButtonElement = document.querySelector('.btn-menu');
+  const closeButtonElement = document.querySelector('.btn-close');
+  const menuLinkElements = document.querySelectorAll('.nav-item');
+
+  // ░█▀▀▀ █──█ █▀▀▄ █▀▀ ▀▀█▀▀ ─▀─ █▀▀█ █▀▀▄ █▀▀
+  // ░█▀▀▀ █──█ █──█ █── ──█── ▀█▀ █──█ █──█ ▀▀█
+  // ░█─── ─▀▀▀ ▀──▀ ▀▀▀ ──▀── ▀▀▀ ▀▀▀▀ ▀──▀ ▀▀▀
```

Comment:
[OPTIONAL] Although these comments look cool, I think I would avoid using them or abusing the use of them as they are not the more legible comments and this could look unprofessional to a certain range of developers.

Reply:
Thank you for the advice!